### PR TITLE
[linstor] update controller version to 1.24.1, update drbd version to 9.2.5

### PR DESCRIPTION
**Pull request by duckhawk**
## Description

Update controller version to 1.24.1
Update DRBD version to 9.2.5
Remove the patches "Add SELinux params for enforced mode" and "Bump k8s api versions and client for piraeus operator" as they are already at upstream.

## Why do we need it, and what problem does it solve?

Currently, there are issues with scheduled backups, as well as several problems within the controller itself.
In DRBD, there is several problems with kernel deadlocks.

## What is the expected result?

Updated linstor controller, correct work of  backup scheduling, and more stable controller
Deadlocks in the kernel caused by the DRBD module will no longer occur.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: linstor
type: feature
summary: Update linstor controller version to 1.24.1. Update DRBD version to 9.2.5.
impact: Linstor controller and piraeus operator restart
impact_level: high
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/static'...

**Comment by z9r5** [comment]
/e2e/abort static;Static;containerd;1.25 5892012105 5892012105-1-con-1-25 /sys/deckhouse-oss/install:pr5469 astra@192.168.199.252

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Review by AleksZimin — DISMISSED** [review]

File: modules/007-registrypackages/images/drbd/werf.inc.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
-{{- $version := "9.2.4" }}
+{{- $version := "9.2.5" }}
 {{- $image_version := $version | replace "." "-" }}
```
Comment (AleksZimin) [inline modules/007-registrypackages/images/drbd/werf.inc.yaml:2]:
```suggestion
```

File: modules/041-linstor/templates/nodegroupconfiguration-drbd-install-centos-like.yaml [hunk 2]
```diff
@@ -51,11 +51,11 @@ spec:
     #   kernel-modules-core   --> kernel-devel
     #   kernel-lt             --> kernel-lt-devel
     #   kernel-lt-core        --> kernel-lt-devel
-    package_name="$(rpm -qf --qf "%{NAME}" "/lib/modules/${version_in_use}" | sed 's/\(-modules-core\)\?$//g; s/\(-core\)\?$/-devel/g')"
+    package_name="$(rpm -qf --qf "%{NAME}" "/lib/modules/${kernel_version_in_use}" | sed 's/\(-modules-core\)\?$//g; s/\(-core\)\?$/-devel/g')"
     bb-yum-install "${package_name}-${version_in_use}"
```
Comment (AleksZimin) [inline modules/041-linstor/templates/nodegroupconfiguration-drbd-install-centos-like.yaml]:
```suggestion
    bb-yum-install "${package_name}-${kernel_version_in_use}"
```

**Review by AleksZimin — APPROVED** [review]

**Review by AleksZimin — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @AleksZimin. I've started the workflow for label 'e2e/run/static'...

**Review by nabokihms — APPROVED** [review]

**Pull request merged**